src/NumericalAlgorithms/Spectral/ComplexDataView.cpp

```diff
@@ -10,7 +10,7 @@
 #include "Utilities/Math.hpp"  // IWYU pragma: keep
 
 namespace Spectral {
-namespace SWSH {
+namespace Swsh {
```

Comment:
I'm going to opt to leave this one as it is, because I think it is more consistent with our style guidelines, treating the abbreviation as a single 'word', and I'm not certain whether or not our other namespaces have been re-examined for consistency with this.